assembl/static2/js/app/components/form/utils.js

```diff
@@ -87,4 +90,20 @@ export function getFileVariable(img: FileValue, initialImg: ?FileValue): FileVar
   // we need to send image: null if we didn't change the image.
   const variab = img && img.externalUrl instanceof File ? img.externalUrl : null;
   return variab;
-}
\ No newline at end of file
+}
+
+export const addEnumSuffixToModuleTitles = (modules: Array<LandingPageModule>): Array<LandingPageModule> => {
```

Comment:
Sorry, I don't like it here. `form/utils` contains only `react-final-form` utils. This code is too specific (really tightly bound to landing page modules). I was thinking about the main `utils` folder but as I said, to move it there, you need to do lots of changes to make it generic. I suggest that you move it in its own file `addEnumSuffixToModuleTitle` in `administration/landingPage/`. Thanks!

Reply:
and don't forget to change the import ;-)